lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.ts

```diff
@@ -192,6 +198,14 @@ export class CardViewTextItemComponent extends BaseCardView<CardViewTextItemMode
         }
     }
 
+    undoText(event: KeyboardEvents) {
+        if ((event.ctrlKey || event.metaKey) && event.code === 'KeyZ') {
+            if (this.textInput.value) {
```

Comment:
I think you can combine that condition with above 'if' using &&

Reply:
i have shorten the code now by combining 2 if's